driver-core/src/test/functional/com/mongodb/client/model/AggregatesFunctionalSpecification.groovy

```diff
@@ -787,6 +793,42 @@ class AggregatesFunctionalSpecification extends OperationFunctionalSpecification
         helper?.drop()
     }
 
+    @IgnoreIf({ !serverVersionAtLeast(4, 3) })
+    def '$accumulator'() {
+        given:
+        def helper = getCollectionHelper()
+        def registry = CodecRegistries.fromProviders(new BsonValueCodecProvider(),
+                new IterableCodecProvider(),
+                new ValueCodecProvider(),
+                new DocumentCodecProvider())
+
+        when:
+        helper.drop()
+        helper.insertDocuments(Document.parse('{_id: 8751, title: "The Banquet", author: "Dante", copies: 2}'),
+                Document.parse('{_id: 8752, title: "Divine Comedy", author: "Dante", copies: 1}'),
+                Document.parse('{_id: 8645, title: "Eclogues", author: "Dante", copies: 2}'),
+                Document.parse('{_id: 7000, title: "The Odyssey", author: "Homer", copies: 10}'),
+                Document.parse('{_id: 7020, title: "Iliad", author: "Homer", copies: 10}'))
+        def initFunction = 'function() { return { count : 0, sum : 0 } }';
+        def accumulateFunction = 'function(state, numCopies) { return { count : state.count + 1, sum : state.sum + numCopies } }';
+        def mergeFunction = 'function(state1, state2) { return { count : state1.count + state2.count, sum : state1.sum + state2.sum } }';
+        def finalizeFunction = 'function(state) { return (state.sum / state.count) }';
+        def accumulatorExpression = accumulator(initFunction, null, accumulateFunction, ['$copies' ],
```

Comment:
Figure a way to use a non-null value for initArgs, so that there is some integration test coverage of that parameter

Reply:
I have added tests that use non-null values for `initArgs` and `accumulateArgs`.